### ISSUE TITLE
Create AIML Support 

### ISSUE THREAD

**Issue by alvin-reyes** [Aurasphere/facebot#18]
We need to incorporate PROGRAM-AB to support AIML as input.

**Comment by Aurasphere** [comment]
I'd like to know more about this and have this in FaceBot. My only concern is: should we do this on the main project or should we implement this as a plugin?

**Comment by alvin-reyes** [comment]
We can separate it as a plugin, but I don't see any reason why we should.
AIML is the standard markup text for voice and chat. Even NLP frameworks
are also incorporating this on their engine. Facebot should have a way to
parse and make a response out of these.

I forked the PROGRAM-AB (ALICE - the official Java implementation/parser of
AIML) cause I want to create a cleaner version of it. We can decide on how
we will implement it when it's ready.

**Comment by Aurasphere** [comment]
Yeah, the plugin will only be necessary if the implementation is heavyweight. If we are talking only about supporting something that is already implemented we will probably end up doing it on the core project. Thank you!

**Comment by alvin-reyes** [comment]
https://github.com/BotMill/aiml-plugin
